api-swagger.json

```diff
@@ -1 +1 @@
":"/oauth/mozilla-subscriptions/customer/plan-eligibility/{planid}","operationId":"getOauthMozillasubscriptionsCustomerPlaneligibilityPlanid","description":"🔒 Authenticated with OAuth bearer token\n\nGet eligibility for a given plan. Returns eligibility as 'create'|'upgrade'|'downgrade'|'blocked_iap'|'invalid'.","parameters":[{"type":"string","description":"A unique identifier for the [plan](https://stripe.com/docs/api/plans/object).","maxLength":255,"name":"planId","in":"path","required":true}],"tags":["Subscriptions"],"responses":{"default":{"schema":{"type":"string"},"description":"Successful"}}}},"/oauth/subscriptions/iap/plans/{appName}":{"get":{"summary":"/oauth/subscriptions/iap/plans/{appName}","operationId":"getOauthSubscriptionsIapPlansAppname","description":"Returns available plans for In-App Purchase clients.","parameters":[{"type":"string","name":"appName","in":"path","required":true}],"tags":["Subscriptions"],"responses":{"default":{"schema":{"type":"string"},"description":"Successful"}}}},"/authorized-clients":{"post":{"summary":"/v1/authorized-clients","operationId":"postAuthorizedclients","description":"This endpoint returns a list of all OAuth client instances connected to the user's account, including the the scopes granted to each client instance and the time at which it was last active, if available. It must be authenticated with an identity assertion for the user's account.","parameters":[{"in":"body","name":"body","schema":{"$ref":"#/definitions/Model39"}}],"tags":["OAuth Server API Overview"],"responses":{"200":{"schema":{"$ref":"#/definitions/Model42"},"description":"A valid 200 response will be a JSON array.\n\nFor clients that use refresh tokens, each refresh token is taken to represent a separate instance of that client and is returned as a separate entry in the list, with the `refresh_token_id` field distinguishing each.\n\nFor clients that only use access tokens, all active access tokens are combined into a single entry in the list, and the `refresh_token_id` field will not be present.\n\n**Example:**\n``` js\n    [\n      {\n        \"client_id\": \"5901bd09376fadaa\",\n        \"refresh_token_id\": \"6e8c38f6a9c27dc0e4df698dc3e3e8b101ad6d79e87842b1ca96ad9b3cd8ed28\",\n        \"name\": \"Example Sync Client\",\n        \"created_time\": 1528334748000,\n        \"last_access_time\": 1528334748000,\n        \"scope\": [\"profile\", \"https://identity.mozilla.com/apps/oldsync\"]\n      },\n      {\n        \"client_id\": \"5901bd09376fadaa\",\n        \"refresh_token_id\": \"eb5e17f246a6b0937356412118ea12b67a638232d6b376e2511cf38a0c4eecf9\",\n        \"name\": \"Example Sync Client\",\n        \"created_time\": 1528334748000,\n        \"last_access_time\": 1528334834000,\n        \"scope\": [\"profile\", \"https://identity.mozilla.com/apps/oldsync\"]\n      },\n      {\n        \"client_id\": \"23d10a14f474ca41\",\n        \"name\": \"Example Website\",\n        \"created_time\": 1328334748000,\n        \"last_access_
```

Comment:
Also, doubt this is anywhere near top 1000 in priority, but looks like possibly 21 instances of "Firefox Accounts" in this PR (although some deleted+readed, so might be half that number, at best)

```js
{"title":"Firefox Accounts API Documentation"
```